exercises/fizz-buzz/test_fizz-buzz.R

```diff
@@ -0,0 +1,125 @@
+source("./fizz-buzz.R")
+library(testthat)
+
+ans10 <- "1
+2
+Fizz
+4
+Buzz
+Fizz
+7
+8
+Fizz
+Buzz"
+  
+ans100 <- "1
+2
+Fizz
+4
+Buzz
+Fizz
+7
+8
+Fizz
+Buzz
+11
+Fizz
+13
+14
+Fizz Buzz
+16
+17
+Fizz
+19
+Buzz
+Fizz
+22
+23
+Fizz
+Buzz
+26
+Fizz
+28
+29
+Fizz Buzz
+31
+32
+Fizz
+34
+Buzz
+Fizz
+37
+38
+Fizz
+Buzz
+41
+Fizz
+43
+44
+Fizz Buzz
+46
+47
+Fizz
+49
+Buzz
+Fizz
+52
+53
+Fizz
+Buzz
+56
+Fizz
+58
+59
+Fizz Buzz
+61
+62
+Fizz
+64
+Buzz
+Fizz
+67
+68
+Fizz
+Buzz
+71
+Fizz
+73
+74
+Fizz Buzz
+76
+77
+Fizz
+79
+Buzz
+Fizz
+82
+83
+Fizz
+Buzz
+86
+Fizz
+88
+89
+Fizz Buzz
+91
+92
+Fizz
+94
+Buzz
+Fizz
+97
+98
+Fizz
+Buzz"
+
+test_that("Vector of strings match", {
+  expect_identical(fizz_buzz(10), cat(ans10) )
+  expect_identical(fizz_buzz(100), cat(ans100) )
```

Comment:
We [don't use `cat` anywhere else](https://github.com/exercism/r/search?q=cat&unscoped_q=cat). How about converting those two tests to the following?

```r
ans10 <- c("1", "2", "Fizz", "4", "Buzz", "Fizz", "7", "8", "Fizz", "Buzz")

ans100 <- c(
  "1", "2", "Fizz", "4", "Buzz", "Fizz", "7", "8", "Fizz", "Buzz", "11", "Fizz",
  "13", "14", "Fizz Buzz", "16", "17", "Fizz", "19", "Buzz", "Fizz", "22", "23",
  "Fizz", "Buzz", "26", "Fizz", "28", "29", "Fizz Buzz", "31", "32", "Fizz",
  "34", "Buzz", "Fizz", "37", "38", "Fizz", "Buzz", "41", "Fizz", "43", "44",
  "Fizz Buzz", "46", "47", "Fizz", "49", "Buzz", "Fizz", "52", "53", "Fizz",
  "Buzz", "56", "Fizz", "58", "59", "Fizz Buzz", "61", "62", "Fizz", "64",
  "Buzz", "Fizz", "67", "68", "Fizz", "Buzz", "71", "Fizz", "73", "74",
  "Fizz Buzz", "76", "77", "Fizz", "79", "Buzz", "Fizz", "82", "83", "Fizz",
  "Buzz", "86", "Fizz", "88", "89", "Fizz Buzz", "91", "92", "Fizz", "94",
  "Buzz", "Fizz", "97", "98", "Fizz", "Buzz"
)
```

Also, shortening (to the second `Fizz Buzz` maybe) should be fine, no?